src/docs/sdk/features.mdx

```diff
@@ -179,3 +179,15 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - HTTP request must be enhanced with a [`sentry-trace` HTTP header](/sdk/performance/#header-sentry-trace)
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
+
+## Attaching Request Body
+
+Ability for the SDK to attach request body to events and/or transactions triggered during the execution of request.
```

Comment:
```suggestion
Ability for the SDK to attach request body to events and triggered during the execution of request.
```

Reply:
While technically it is possible to set it on transactions, no SDK does it at this stage.